src/Oro/Bundle/DataGridBundle/Extension/Pager/Orm/Pager.php

```diff
@@ -139,24 +138,28 @@ public function init()
 
         /** @var QueryBuilder $query */
         $query = $this->getQueryBuilder();
+        $countQb = $this->countQueryBuilderOptimizer->getCountQueryBuilder($this->getQueryBuilder());
 
         $query->setFirstResult(null);
         $query->setMaxResults(null);
 
+
         if (count($this->getParameters()) > 0) {
             $query->setParameters($this->getParameters());
```

Comment:
Pager can receive additional parameters, it looks like $countQb should consider them too